docs/tutorials/services/constrain-motion.md

```diff
@@ -0,0 +1,432 @@
+
+If you are using a `fake` gripper, there is no real hardware to calibrate and you can [continue this tutorial](#use-a-transform-to-represent-a-drinking-cup), imagining that your fake gripper corresponds to the diagram above.
+
+If you are using a real arm and gripper, use the **Control** tab in the [Viam app](https://app.viam.com/) to move the gripper, look at its reported orientations, and map them to its orientation in the real world.
+If the axes are different from those described above, take these differences into account in your code.
+
+## Use a transform to represent a drinking cup
+
+Imagine your cup is 120 millimeters tall with a radius of 45 millimeters.
+You need to take this space into account to avoid bumping objects on the table with the cup.
+
+You can pass transforms to the [Motion Service `move` method](../../../services/motion/#move) to represent objects that are connected to the robot but are not actual robotic components.
+To represent the drinking cup held in your robot's gripper, add this to your code:
+
+```python {class="line-numbers linkable-line-numbers"}
+# Create a transform to represent the cup as a 120mm tall, 45mm radius capsule shape
+cup_geometry = Geometry(center=Pose(x=0, y=0, z=155),
+    capsule=Capsule(radius_mm=45, length_mm=120))
+transforms = [
+    # Name the reference frame "cup" and point its long axis along the y axis of the gripper
+    Transform(reference_frame="cup", pose_in_observer_frame=PoseInFrame(
+        reference_frame="myGripper",
+        pose=Pose(x=0, y=0, z=45, o_x=0, o_y=-1, o_z=0, theta=0)),
+        physical_object=cup_geometry)
+]
+```
+
+Here, we use a 155mm offset along the gripper's Z axis to represent the distance from the base of the gripper to the center of the cup.
+We assume the cup is situated against the palm of the gripper's hand.
+This offset is then equal to the radius of the cup (45mm) plus the distance from the gripper's origin (where it meets the arm) to its palm (110mm in this example).
+If your gripper has different dimensions, change the offset accordingly.
+
+Now that you have created the table and tissue box obstacles and the cup transform, create a [WorldState](https://python.viam.dev/autoapi/viam/proto/common/index.html#viam.proto.common.WorldState) that includes all of them:
+
+```python {class="line-numbers linkable-line-numbers"}
+# Create a WorldState that includes the table and tissue box obstacles, and the cup transform
+world_state = WorldState(obstacles=[obstacles_in_frame], transforms = transforms)
+```
```

Comment:
I'm leaving it in the full tutorial code though, so I think I'll leave it in here so that it's matchy-matchy and people can refer back and forth more easily as they build out there full code file. For me, when going through the second tutorial, I found that to be useful.

Reply:
Makes sense to me!